tools/appliance/definitions/systemvmtemplate/configure_networking.sh

```diff
@@ -52,8 +45,24 @@ function configure_hostname() {
   hostname $HOSTNAME
 }
 
+function configure_interfaces() {
+  cat > /etc/network/interfaces << EOF
+source /etc/network/interfaces.d/*
+
+# The loopback network interface
+auto lo
+iface lo inet loopback
+
+# The primary network interface
+auto eth0
+iface eth0 inet dhcp
+pre-up sleep 2
```

Comment:
Historic reasons, I've no idea. I simply moved the code from here: https://github.com/apache/cloudstack/blob/master/tools/appliance/definitions/systemvmtemplate/configure_networking.sh#L28